app/vmui/packages/vmui/src/components/Chart/Line/Legend/LegendConfigs/LegendConfigs.tsx

```diff
@@ -32,12 +39,21 @@ const LegendConfigs: FC<Props> = ({ data }) => {
     onChangeView(value);
   };
 
+  const handleOpenSettings = () => {
+    graphDispatch({ type: "SET_OPEN_SETTINGS", payload: true });
+  };
+
   return (
-    <>
+    <div
+      className={classNames({
```

Comment:
It would be great to add help tooltips on hover with a brief description of what it does.